lib/src/rules/length.ts

```diff
@@ -0,0 +1,29 @@
+export interface LengthParams {
+  length: number;
+}
+
+export function parseLengthParams(customParams: LengthParams, errorMessage: string) {
+  const length = customParams.length === null ? NaN : Number(customParams.length);
```

Comment:
If it is undefined it is casted with `Number()` and getting NaN. I asked for `null` and did not included  `undefined` because when you do a `Number(null)` you get `0` instead of `NaN`.